src/main/kotlin/Config.kt

```diff
@@ -51,9 +55,15 @@ object Config {
             put("value.serializer", ByteArraySerializer::class.java)
 
             put("auto.offset.reset", "earliest")
+            put(metaDataRefreshKey, getEnv("K2HB_KAFKA_META_REFRESH_MS") ?: "10000")
         }
 
-        val pollTimeout: Duration = getEnv("K2HB_KAFKA_POLL_TIMEOUT")?.toDuration() ?: Duration.ofDays(10)
-        val topics = getEnv("K2HB_KAFKA_TOPICS")?.split(',') ?: listOf("test-topic")
+        val pollTimeout: Duration = getEnv("K2HB_KAFKA_POLL_TIMEOUT")?.toDuration() ?: Duration.ofMinutes(10)
```

Comment:
This doesn't match the readme, which says:

_It will poll the `test-topic` topic with a poll timeout of 10 **days**, and refresh the topics list every 10 seconds_

Reply:
Done